### PR TITLE
Feat: create child note

### PR DESCRIPTION
**Pull request by kloV148**
Added ability to create child note

### CODE REVIEW

File: src/application/router/routes.ts [hunk 1]
```diff
@@ -40,6 +40,15 @@ const routes: RouteRecordRaw[] = [
       id: null,
     },
   },
+  {
+    name: 'test',
```
Comment (neSpecc) [inline src/application/router/routes.ts:44]: strange name

File: src/infrastructure/note.repository.ts [hunk 2]
```diff
@@ -61,9 +61,11 @@ export default class NoteRepository implements NoteRepositoryInterface {
    * Creates a new note
    *
    * @param content - Note content (Editor.js data)
+   * @param parentId - Id of the parent note. If null, then it's a root note
    */
-  public async createNote(content: NoteContent): Promise<Note> {
-    const response = await this.transport.post<{ id: NoteId }>('/note', { content });
+  public async createNote(content: NoteContent, parentId: NoteId | null): Promise<Note> {
+    const response = await this.transport.post<{ id: NoteId }>('/note', { content: content,
+      parentId: parentId });
```
Comment (neSpecc) [inline src/infrastructure/note.repository.ts:68]:
```suggestion
    const response = await this.transport.post<{ id: NoteId }>('/note', {
      content,
      parentId,
    });
```

File: src/presentation/pages/Note.vue [hunk 3]
```diff
@@ -1,30 +1,46 @@
...
 
+const router = useRouter();
+
 const props = defineProps<{
   /**
    * Null for new note, id for reading existing note
    */
   id: string | null;
+
+  parentId: string | null;
```
Comment (neSpecc) [inline src/presentation/pages/Note.vue:43]: jsdoc is missed

File: src/presentation/pages/Note.vue [hunk 4]
```diff
@@ -1,30 +1,47 @@
...
-    v-else
-    ref="editor"
-    :content="note.content"
-    :read-only="!canEdit"
-    @change="noteChanged"
-  />
+  <div v-else>
+    <div class="control__button">
+      <Button
+        class="header__plus"
```
Comment (neSpecc) [inline src/presentation/pages/Note.vue:6]: unexpected class

File: src/presentation/pages/Note.vue [hunk 5]
```diff
@@ -1,30 +1,47 @@
...
+        text="createChildNote"
```
Comment (neSpecc) [inline src/presentation/pages/Note.vue:7]:
```suggestion
        text="Add child note"
```

File: src/presentation/pages/Note.vue (hunk 4 above)
Comment (kloV148) [reply src/presentation/pages/Note.vue:6]: Maybe just don't specify classes for now?

Comment (neSpecc) [reply src/presentation/pages/Note.vue:6]: if you don't use it, clear it

**Review by neSpecc — APPROVED** [review]

**Pull request merged**